### ISSUE TITLE
Import for report files could parse incorrectly XMLs without any output[v5]

### ISSUE THREAD

**Issue by arybkin** [reportportal/reportportal#1241]
**Describe the bug**
May be a feature. When you import zip files with xml reports, it tries to parse it and ignores any unknown tags and doesn't inform you, that something was unclear

**To Reproduce**
1. Upload report for CPP boost tests with any failed test
[Project_CPP_BOOST_WITH_EXCEPTION.zip](https://github.com/reportportal/reportportal/files/5638884/Project_CPP_BOOST_WITH_EXCEPTION.zip)
2. Click import button -> Report is SUCCESSFULLY imported with 19 passed tests. But in report there is one failed tests with tag Exception, that is unknown for report portal

**Expected behavior**
Report Portal returns Yellow (partially succeeded) with output, what tags/parts weren't parsed correctly. This output should be available on UI and via API

**Screenshots**
![image](https://user-images.githubusercontent.com/6349907/101087963-08a7aa00-35c4-11eb-9b1a-fafc42539bb8.png)

**Desktop (please complete the following information):**
 - OS: Win10
 - Browser FF
 - Version 83.0

**Additional context**
Basically it could be just a lack of functionality. Please move it to feature, if you think so

**Comment by evjlobanova** [comment]
For now, attributes are not imported to ReportPortal.
Import is a limited feature of RP, our main direction is real-time reporting.
So that for now we do not plan to improve import feature.

**Comment by evjlobanova** [comment]
For now the system just validates if a file is xml and have junit format

**Comment by MaryiaRymarchyk** [comment]
@arybkin  it is possible to send one attribute  if you provide within the URL during import. Example : {{url}}/{{project}}/launch/import?attributeKey=attributeKey1&attributeValue=attributeValue1
However with the nearest release 24.1 (we expect it soon in the early of Feb) - it would be possible to send several attributes using message body.
![Import with attributes](https://github.com/reportportal/reportportal/assets/122868739/3e565890-a828-4fc4-ad01-a7a0b8407abe)

Example of   launchImportRq : ```
{"attributes": [   {
    "key": "AttributeKeyI",
    "system": false,
    "value": "AttributeValueI"},
   { "key": "AttributeKeyII",
    "system": false,
    "value": "AttributeValueII"}],
  "description": "MB Test Description for .xml imported launch with 2 attributes",
  "mode": "DEFAULT",
  "name": "MB .xml Launch, 2 attributes",
  "startTime": "2017-05-25T14:15:01"
}
```

Within the release new documentation with the guide how to do it will be available.

**Comment by Kozovskaya** [comment]
Hello @arybkin 
I'm pleased to let you know that the fix is already released!
https://github.com/reportportal/reportportal/releases/tag/v24.1
